software/excalidraw.yml

```
@@ -0,0 +1,10 @@
+name: excalidraw
+website_url: https://excalidraw.com
+description: Virtual whiteboard for sketching hand-drawn like diagrams
+licenses:
+  - MIT
+platforms:
+  - Typescript
+tags:
+  - Miscellaneous
```

> https://awesome-selfhosted.net/#wbo

A similar whiteboard tool is here.

```suggestion
  - Communication - Custom Communication Systems
```